tests/taskcluster/bin/raptor_graph

```diff
@@ -23,6 +23,9 @@ var originalHeadRev = process.env.GITHUB_HEAD_REV;
 
 const TC_CLIENT_OPTS = { timeout: 30 * 1000 };
 
+// taskcluster worker type for raptor tasks
+RAPTOR_WORKER_TYPE = 'raptor-single-capacity';
```

Comment:
So it probably doesn't matter, but the decision task and the results comparison could probably be on an instance that can do more concurrent tasks.